docs/content/vectorising.md

```diff
@@ -11,7 +11,7 @@ title: Vectorising your environments
 Gymnasium provides two types of vectorized environments:
 
 - `gymnasium.vector.SyncVectorEnv`, where the different copies of the environment are executed sequentially.
```

Comment:
With the new line below, should not this not be removed

Reply:
Fixed